src/Adapter/Feature/MultistoreFeature.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Feature;
+
+use PrestaShop\PrestaShop\Adapter\Entity\Shop;
+use PrestaShop\PrestaShop\Core\ConfigurationInterface;
+use PrestaShop\PrestaShop\Core\Feature\FeatureInterface;
+
+/**
+ * Class MultishopFeature provides data about multishop feature usage.
+ *
+ * @internal
+ */
+final class MultistoreFeature implements FeatureInterface
+{
+    /**
+     * @var ConfigurationInterface
+     */
+    private $configuration;
+
+    /**
+     * @param ConfigurationInterface $configuration
+     */
+    public function __construct(ConfigurationInterface $configuration)
+    {
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isUsed()
+    {
+        // internally it checks if feature is active
+        // and at least 2 shops exist
+        return Shop::isFeatureActive();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isActive()
+    {
+        return (bool) $this->configuration->get('PS_MULTISHOP_FEATURE_ACTIVE');
```

Comment:
Too bad the interface only provides `get` & `set`, I was going to suggest using `getBoolean`.

Reply:
this is what we tried to do with this PR https://github.com/PrestaShop/PrestaShop/pull/9193 but its stuck, nobody reviews it. -_-

Reply:
Rely on configuration adapter instead please :+1:

Reply:
i will have to. :(